src/wirecloud/platform/static/js/wirecloud/ui/WiringEditor.js

```diff
@@ -43,7 +43,7 @@ Wirecloud.ui = Wirecloud.ui || {};
      *      [TODO: description]
      */
     ns.WiringEditor = function WiringEditor(id, options) {
-        options = utils.updateObject({}, options);
```

Comment:
Why are those `updateObject` to `merge` replacements being added to this commit? I'm not going to stop this pull request by those changes but... they don't seem related to removing the `defineClass` util...